For some reason, unciv seems to always try and build the android version.

Follow the AUR's fix and remove the android section completely.

Also fixes errors relating to robovm dependency by removing the ios build.

--- build.gradle.kts.original	2023-03-11 19:18:53.000000000 +0000
+++ build.gradle.kts	2023-03-11 19:19:24.000000000 +0000
@@ -80,36 +80,6 @@
 
 }
 
-project(":android") {
-    apply(plugin = "com.android.application")
-    apply(plugin = "kotlin-android")
-
-    val natives by configurations.creating
-
-    dependencies {
-        "implementation"(project(":core"))
-        "implementation"("com.badlogicgames.gdx:gdx-backend-android:$gdxVersion")
-        "implementation"("org.jetbrains.kotlinx:kotlinx-coroutines-android:1.6.4")
-        natives("com.badlogicgames.gdx:gdx-platform:$gdxVersion:natives-armeabi-v7a")
-        natives("com.badlogicgames.gdx:gdx-platform:$gdxVersion:natives-arm64-v8a")
-        natives("com.badlogicgames.gdx:gdx-platform:$gdxVersion:natives-x86")
-        natives("com.badlogicgames.gdx:gdx-platform:$gdxVersion:natives-x86_64")
-    }
-}
-
-project(":ios") {
-    apply(plugin = "kotlin")
-    apply(plugin = "robovm")
-
-    dependencies {
-        "implementation"(project(":core"))
-        "implementation"("com.mobidevelop.robovm:robovm-rt:$roboVMVersion")
-        "implementation"("com.mobidevelop.robovm:robovm-cocoatouch:$roboVMVersion")
-        "implementation"("com.badlogicgames.gdx:gdx-backend-robovm:$gdxVersion")
-        "implementation"("com.badlogicgames.gdx:gdx-platform:$gdxVersion:natives-ios")
-    }
-}
-
 
 project(":core") {
     apply(plugin = "kotlin")
